### ISSUE TITLE
1. Make CLDF compliant annotated dataset for the Cāng Jiépiān 蒼頡篇

### ISSUE THREAD

**Issue by nh36** [hanproj/bamboo-and-wood#3]
The work for the 蒼頡篇 is largely done: https://calc.hypotheses.org/2525

**Comment by LinguList** [comment]
Yes, I'd argue also: we can ignore the more finegrained annotation attempts made in this spreadsheet and first get them in line with our models. THEN, of course, one can discuss to which degree one can reflect parts of the models Chris had in mind.

**Comment by nh36** [comment]
Mattis has made a model using Baxter of a CLDF complaint dataset: https://github.com/hanproj/baxterocrhymes

Our next goal is to look at basic formats for unannotated rhyme collections.
